UPGRADING.md

```diff
@@ -84,7 +84,7 @@ Interceptors behave very similarly to outer around hooks, with the following cha
 An base interceptor looks like this:
 
 ```ruby
-class MyInterceptor < ::Gruf::Interceptors::ServerInterceptor
```

Comment:
Is there a reason for all of these removals? Generally I prefer to be explicit with inheritance so as to prevent namespace collisions with external gems.

Reply:
Firstly, I think the code will look better if we can remove all global scope resolution `::` 
But I'm not 100% sure about conflicting. So I reverted all of them.